### ISSUE TITLE
Feature Request: cmd + left click on file shortcut, to open file in split screen like Apple's Xcode

### ISSUE THREAD

**Issue by chewedon** [adobe/brackets#11308]
Hello Adobe.

Really like Brackets IDE, very minimal, good job! 

My only complaint is the split screen feature isn't working as intuitive as I'd hope.

Left clicking on a file while the right hand side panel is focused still opens the file on the left hand side  panel instead of the right hand side panel.

Is Adobe able to make it to so that I hold Cmd + Left Click on a file in the project tree and Bracket automatically opens a split view showing the file I Cmd + Left Clicked on ?

A normal left click without holding the Cmd key on the keyboard, can simply open the file in the left hand side panel.

This is how Xcode does it. It is very elegant way to do it and not prone to error.

**Comment by petetnt** [comment]
:+1: this. Opening new files to the "wrong" split view I was aiming at is a daily grievance for me.

How do other editors besides XCode handle this?

**Comment by nethip** [comment]
@chewedon @petetnt I am not quite sure if I understand the problem correctly. In the split view whichever is the focused pane, the new files will be loaded to that pane. The files then can be dragged in the Working Set to move files across the panes? am I missing something here?

**Comment by petetnt** [comment]
@nethip Can't speak for @chewedon but for me it has been a long running usability issue where much of the problems spawn from the fact that it's sometimes hard to see which pane is focused and which one is not.

Consider this:

1. I want to open `foo.js` to the left pane
2. I click the file in the project tree and the file opens to right pane as I had it focused
3. Now I have to either a) Go to the working set, switch to an arbitrary file, focus the left pane, open the file again. (or if the right side working set is empty, toggle the split view to make the file "available" again) b) Make a change to `foo.js` so I can drag it across the working set (which most often is unwanted)

If I could just decide on which pane I wanted the file to open without worrying about working sets or focused panes, I'd run to those kinds of small but sometimes frustrating issues. (However, the working set trickery on switching around opened but not modified files would still be there).

**Comment by chewedon** [comment]
@nethip,

The problem for me is when I open a file say `foo.js` in the left hand side panel. Then some time later, I want to open `foo.js` in the right hand side panel, it changes the focus back to the left hand side and opens `foo.js` there because `foo.js` was first opened in the left hand side panel.

I am aware I can drag the file in the "working sets" area but that feels too long and painful to have to manually manage which side of the panel the file should be opened in, compared to simply Cmd + Left Click on the file. The work around I have to do is click on the X to clear the file from the LHS working set, so that I open the file on the RHS panel.

As the user, it should be my choice even if I want to show the same file on both the LHS and RHS panels but I feel Bracket is trying to be smart and deciding thing for me by telling me "hey you've already got it opened on the left hand side so I'm just going to load it in the left hand side". My response is usually "NO! I want it on the right hand side! Just do what I want! argh!!!! *throws computer out the window*"

Microsoft's Visual Studio Code (https://code.visualstudio.com/) has the Cmd + Left Click feature I'm talking about. The only thing holding me back from using Visual Studio Code is that it is in an infant stage (not far behind Brackets), it's just missing the colour highlighting for various programming languages, otherwise I think it's a good alternative.

Xcode actually goes one step further. If you hold Cmd + Alt + Shift + Left CLick on a file, it lets you choose how you want to split the new window using a popup diagram and you tell it the quadrant you want to open the new file in, it does just that, split the window as you've chosen and open the file in that window. I use this when I want to open a ViewController.m file in the LHS full height, and open a Class.h and Class.m equally split vertically on the RHS.

**Comment by MarcelGerber** [comment]
@chewedon I don't own/use a Mac, so maybe that's a dumb question, but isn't Cmd + Click meant to open the context menu?

**Comment by chewedon** [comment]
@MarcelGerber 

Yes, sorry. When I say Cmd + Left Click I meant Alt + Left Click.

My finger remembers the general area of the key I'm pressing not the actual name of the key :D
